README.md

```diff
@@ -66,6 +68,36 @@ may need to change the permissions manually for the generated files after the
 These commands are the ones used by our CI system to ensure that the Protobuf
 files stay semantically valid.
 
+## Build Using Bazel
+
+![build proto](https://github.com/p4lang/p4runtime/workflows/build%20proto/badge.svg)
+
+The protobufs can also be built using [Bazel](https://bazel.build/).
+The Bazel WORKSPACE and BUILD files are located in the [proto folder](proto/).
```

Comment:
Add files extensions to these (.bazel)?

Reply:
Since the audience here is Bazel users, I felt that adding .bazel was more likely to cause confusion than to help, since it's not generally used in the Bazel documentation. In particular, I'm using "WORKSPACE file" here to refer to the Bazel concept of WORKSPACE files, rather than the actual filename (much like a README file is a README file regardless of whether it has an .txt or .md extension). Let me know if that makes sense; I'm open to changing it if you feel strongly.